include/magma/Swapchain.hpp

```diff
@@ -64,7 +64,8 @@ namespace magma
       Swapchain(Surface const &surface,
                 magma::Device<claws::no_delete> device,
                 vk::PhysicalDevice physicalDevice,
-                claws::handle<Swapchain, claws::no_delete> old)
+                claws::handle<Swapchain, claws::no_delete> old,
+		vk::Extent2D desiredSize = vk::Extent2D{0xFFFFFFFFu, 0xFFFFFFFFu})
```

Comment:
What is theses addresses or hexadecimal codes ? :O

Reply:
They're an extension of the special values in the spec, which explicitly uses them for cases where the size is undefined. I decided to imitate that somewhat.
These are not addresses, they are the maximum value of an uint32_t.